adserver/templates/adserver/advertiser/overview.html

```diff
@@ -62,11 +62,51 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
     </div>
     <p class="text-right mb-5"><a href="{% url 'advertiser_report' advertiser.slug %}?start_date={{ start_date|date:'Y-m-d' }}&end_date={{ end_date|date:'Y-m-d' }}">{% trans 'detailed reporting' %} &raquo;</a></p>
 
-    <div class="row mb-5">
-      <div class="col min-vh-50">
-        {% metabase_question_embed metabase_advertiser_performance advertiser_slug=advertiser.slug start_date=start_date end_date=end_date %}
+    {% if advertiser_new %}
+      {# Advertiser onboarding workflow #}
+      <div class="row mb-5">
+        <div class="col">
+          <h3>{% trans 'There are a few steps to getting started with your first ad campaign with us.' %}</h3>
+          <ol class="mt-5 lead">
+            <li>
+              {% url "flight_list" advertiser.slug as flight_list_url %}
+              <p>{% blocktrans %}View the upcoming <a href="{{ flight_list_url }}">flights</a> set up for you and make sure the content and geographic targeting matches your customer focus. The price of your campaign depends on your chosen targeting.{% endblocktrans %}</p>
+            </li>
+            <li>
+              {% url "support" as support_url %}
+              <p>{% blocktrans %}Once your flights are set up, we will issue you an invoice. You can upload your ad creatives to your flights.{% endblocktrans %}</p>
+            </li>
+            <li>
+              <p>{% blocktrans %}Once the invoice is paid, we can start your flight and start sending traffic to you. We can also set your flight to start automatically on a specific date.{% endblocktrans %}</p>
```

Comment:
This is very passive voice with too many `can`'s. It feels a bit like we're trying to explain the process, but also explain additional features? 

I think this copy can be improved to be clearer on what *they* need to do, and probably have less somewhat random feature references? It seems like that should all be handled somewhere else, and we could link to it.

Reply:
I reworded it:
![Screenshot from 2022-05-03 14-14-43](https://user-images.githubusercontent.com/185043/166567607-e050391a-9f46-421f-8342-c0dae92f6278.png)